_sources/governance.rst.txt

```diff
@@ -101,17 +101,17 @@ The Chair will be appointed by the Steering Council. The Chair can stay on as lo
 
 Release Manager
```

Comment:
@slumnitz @sjsrey Do we also need to decide on a Release Manager or does this automatically fall to the BDFL?

Reply:
I would suggest to keep it separate appointments, but not exclude that someone can have multiple appointments?

Reply:
For the meta package, the one of the maintainers could be the release manager?